node/lib/openshift-origin-node/model/application_container.rb

```diff
@@ -659,6 +660,23 @@ def stop_lock?
         @cartridge_model.stop_lock?
       end
 
+      def start_priority_file
+        return PathUtils.join(@container_dir, '.start_priority')
+      end
+
+      def start_priority
+        if @start_priority.nil? && File.exists?(start_priority_file)
+          @start_priority = File.open(start_priority_file).read.to_i rescue nil
+        end
+        @start_priority ||= DEFAULT_START_PRIORITY
+        return @start_priority
+      end
+
+      def start_priority=(prio)
+        File.new(start_priority_file, File::CREAT|File::TRUNC|File::WRONLY, 0644).write(prio)
```

Comment:
With this File.new() not being in a block (which calls close()) or calling .close(), are we leaking file descriptors?